packages/@aws-cdk/aws-sns-subscriptions/test/subs.test.ts

```diff
@@ -550,3 +550,13 @@ test('with filter policy', () => {
     }
   });
 });
+
+test('region property is present on an imported topic', () => {
```

Comment:
add a test where the imported property takes the topicArn from a CfnParmeter for example:

```ts
const p = new CfnParameter(...);
const topic = sns.Topic.fromTopicArn(this, 'foo', p.valueAsString);
// BOOM
```